src/EA.Weee.Web/Views/Shared/_ViewTransferEvidenceNotePartial.cshtml

```diff
@@ -68,58 +78,86 @@
         }
     </tbody>
 </table>
-<h2 class="govuk-heading-m govuk-!-padding-top-5 govuk-!-margin-bottom-2">Breakdown of evidence transferred</h2>
+
+<hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
+
+<div class="govuk-table govuk-!-margin-bottom-0">
+    <div class="govuk-grid-column-one-half govuk-!-margin-left-0 govuk-!-padding-left-0 govuk-!-margin-bottom-2"><h2 class="govuk-heading-m" style="text-align:left; padding-left:0; margin-left:0;">Breakdown of evidence transferred</h2></div>
```

Comment:
Can use the govuk padding zeros for this :)

Reply:
yes